Formula/bash-completion.rb

```diff
@@ -1,5 +1,5 @@
 # NOTE: version 2.0 is out, but it requires Bash 4, and macOS ships
-# with 3.2.48. See homebrew-versions for a 2.0 formula.
+# with 3.2.48. If you've upgraded bash, use bash-completion@2 instead.
```

Comment:
It doesn't really matter since the key point is that the system bash is <4. However, I'll fix it when I merge.